src/info/info.ts

```diff
@@ -389,6 +394,12 @@ port.onMessage.addListener(async (message) => {
             // event listener for serial comm button
             btn.addEventListener("click", async _ => {
                 // const worker = new Worker(browser.runtime.getURL("./info/worker.js"), { type: "module" });
+                
+                // only show the Arduino Zero
+                const filters = [
```

Comment:
Do we need this? I don't think it is used anywhere

Reply:
yes, we just forgot to use it 😅 I added that to the const hapticPort line below